### ISSUE TITLE
Having a default value on a field overrides the "Last Value" option

### ISSUE THREAD

**Issue by roemhildtg** [Esri/local-government-desktop-addins#266]
For instance, if I set a text field default value to "hello" and then have a Last Value action run on that field, the default value always gets populated and that overrides the Last Value array and value set.

**Comment by roemhildtg** [comment]
Temporary solution: just get rid of the default value - Last Value / AA provides a much more powerful interface so I'm using that instead.

**Comment by MikeMillerGIS** [comment]
I will take a look, but I recall there was not an easy fix as the default value is registering as a field change.

**Comment by roemhildtg** [comment]
Yep, I figured it might be something like that. Not a huge issue.
